android/expoview/src/main/java/versioned/host/exp/exponent/ExponentPackage.java

```diff
@@ -72,6 +73,9 @@
   private final Map<String, Object> mExperienceProperties;
   private final JSONObject mManifest;
 
+  // Shared element package shares a "node manager" between native module and view manager
+  private final RNSharedElementPackage mSharedElementPackage = new RNSharedElementPackage();
```

Comment:
cc @IjzerenHein @sjchmiela - any ideas for a better way to organize this either in react-native-shared-element or here? for reference: https://github.com/IjzerenHein/react-native-shared-element/blob/9c3f1dd08a13900c61d5d40116787d8dc7ae9ad9/android/src/main/java/com/ijzerenhein/sharedelement/RNSharedElementPackage.java

Reply:
that way works for me now, but we may want to refactor it in the future if we want to do customizable adhoc builds in the future

Reply:
I agree. I don't really know a good solution for it though. Do you know whether it's possible to get access to a Module from a ViewManager (or vice versa)? I could then host the NodeManager in one of those classes and get access to that way. Not sure that would be good practise though? 🧐

Reply:
You could use `ReactContext#getNativeModule(Class)` to access `NativeModule` from `ViewManager`. 🤔 

https://github.com/react-native-community/react-native-webview/blob/f24805d091e7e37b85431cd5f9ec771cc3b09d37/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java#L591-L593

Reply:
@IjzerenHein - @sjchmiela's suggestion above might work for you! lmk

Reply:
Awesome, thanks @sjchmiela. I'm gonna check that out, will probably be tomorrow.

Reply:
I refactored the code and the nodeManager now no longer needs to be passed in to the Module and ViewManager. It seems to work well, but I want to do a bit more testing on it. Will release a new version tomorrow.

When do you expect to release 35 btw?

Reply:
our cutoff for native code inclusion is likely tomorrow, so i'll pull in any last changes you make tomorrow :)

Reply:
Alright cool. I just finished testing in all my setups and everything looks good. I published the new version as 0.5.1 (https://github.com/IjzerenHein/react-native-shared-element/releases/tag/v0.5.1)